bookkeeper-server/src/main/java/org/apache/bookkeeper/client/DefaultEnsemblePlacementPolicy.java

```diff
@@ -92,6 +94,9 @@
                     }
                     newBookies.add(b);
                     --ensembleSize;
+                    if (ensembleSize == 0) {
+                        return newBookies;
```

Comment:
simple ```return newBookies;``` after the while loop should be enough.

Reply:
may be, but i just want to follow the existing code format here, like how it is dealt in else block